packages/webdriver/src/request.js

```diff
@@ -141,7 +150,7 @@ export default class WebDriverRequest extends EventEmitter {
             if(error.name === 'stale element reference') {
                 log.warn('Request encountered a stale element - terminating request')
                 this.emit('response', { error })
-                return reject(error)
+                return Promise.reject(error)
```

Comment:
is it required to return promise rejection instead of just throwing an error?

Reply:
in this case I can indeed just throw.